samples/msal-react-samples/typescript-sample/package.json

```diff
@@ -23,7 +23,7 @@
         "start": "react-scripts start",
         "build": "react-scripts build",
         "build:package": "cd ../../../lib/msal-react && npm run build:all",
-        "test": "jest",
+        "test": "jest --runInBand",
```

Comment:
Why the change to runInBand? Won't this slow down our tests?

Reply:
You're right, it would (maybe not much at the moment, since there are 2 test files in each sample). I needed this now that I'm setting the configuration parameters at test runtime, so in principle different tests can have different parameters and running them in parallel would lead to conflict. The inefficiency here is that both files use the same parameters here -I'm looking into doing this as an [environment setup step with jest](https://jestjs.io/docs/configuration#setupfilesafterenv-array), so that we can get rid of `runInBand` afterwards

Reply:
Alternative approach: If a different config is needed we can do it on a per-route basis in the app itself rather than swapping it out in the test. Not only does that allow us to run the tests in parallel but also allows us to easily test the scenario manually or our customers to see a feature demonstrated

Anyway, I think since both files are using the same config right now (and I don't see that changing anytime soon) we can add this flag if and when it becomes needed.